crypto/bindings/conversion.ts

```diff
@@ -98,3 +158,120 @@ function fieldsFromRustFlat(fieldBytes: Uint8Array): MlArray<Field> {
   }
   return [0, ...fields];
 }
+
+// affine
+
+type WasmAffine = wasmNamespace.WasmGVesta | wasmNamespace.WasmGPallas;
+type MakeAffine<A extends WasmAffine> = () => A & { ptr: number };
+
+function affineFromRust(pt: WasmAffine): OrInfinity {
+  if (pt.infinity) {
+    pt.free();
+    return 0;
+  } else {
+    let x = fieldFromRust(pt.x);
```

Comment:
Might be obvious, but still asking.
```
function fieldFromRust(x: Uint8Array): Field {
  return x;
}
```
This does create a branch new value right? i.e. the free on the point won't free the field element?

Reply:
So it is going to access a freed value atm, and you plan to update it later?

Reply:
This function does nothing but later it will do a conversion because on one side the field type will change.
I wouldn't say it accesses a freed value (whatever that might mean for a function that does nothing), because the field value here is already copied out of the wasm memory and is a self-contained array of bytes managed by js

Reply:
So what's creating the new value here is the property access, `pt.x`!